doc-content/enterprise-only/cases/case-management-clean-case-log-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='case-management-clean-case-log-proc']
+= Removing case logs from the database
+
+Use the `CaseLogCleanupCommand` to cleanup cases, such as canceled cases that lack value and that are using database space. The `CaseLogCleanupCommand` command consists of logic to clean up all or selected cases automatically. By default, the `CaseLogCleanupCommand` is executed once a day but can be configured to run on different intervals.
+
+There are several important configuration options that can be used with the `CaseLogCleanupCommand` command:
+
+.CaseLogCleanupCommand parameters table
+[cols="1,1,1", options="header"]
+|===
+| Name
+| Description
+| Is Exclusive
+
+| SkipProcessLog
+| Indicates if the clean-up of process instances, node instances and variables log cleanup should be omitted (default: false)
```

Comment:
"should be omitted" or "is omitted"?  I am guessing that if this parameter is flagged then the process & node instances along with the variable log clean up will be skipped when the command runs.

Same comment throughout.

Reply:
Reworded:
Indicates whether or not the process and node instances, along with the process variable log clean-up will be skipped when the command runs. Default value: `false`